ironfish/src/rpc/routes/wallet/importViewAccount.test.ts

```diff
@@ -0,0 +1,40 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
```

Comment:
Why do you need these eslint disables? We should fix the type error, not disable the linter

Reply:
That snuck in from the test that was used as a template. Removing/fixing.